cron/src/pools.rs

```diff
@@ -22,20 +24,39 @@ pub fn get_scraped_entries(
     let binding = raw_scraped_entries[chain_name][chain_id].clone();
     let parsed_scraped_entries: &Vec<Value> = binding.as_array().unwrap();
     let mut dexes_to_register: HashSet<String> = HashSet::new();
+    let mut lp_assets: HashMap<String, AssetInfoBase<String>> = HashMap::new();
 
     let scraped_entries_vec: Vec<(UncheckedPoolAddress, PoolMetadata)> = parsed_scraped_entries
         .iter()
         .map(|value| {
             let pool: (UncheckedPoolAddress, PoolMetadata) =
                 serde_json::from_value(value.clone()).unwrap();
-
+            if let UncheckedPoolAddress::Id(pool_id) = pool.0 {
+                let dex = &pool.1.dex;
+                let assets = pool
+                    .1
+                    .assets
+                    .iter()
+                    .map(AssetEntry::as_str)
+                    .collect::<Vec<_>>()
+                    .join(",");
+                let asset_name = format!("{dex}/{assets}",);
+                lp_assets.insert(
+                    asset_name,
+                    AssetInfoBase::Native(format!("gamm/pool/{pool_id}")),
```

Comment:
this will do this for every chain, we only want this for osmosis right?

Reply:
Pool ids is only osmosis right now

Reply:
we can do it only for osmosis, if we think other chains that add that type of module will use not gamm

Reply:
I don't see why we would need to register the LP tokens of astroport with a key of osmosis's gamm module.